TypeCobol/Compiler/CodeElements/CodeElementType.cs

```diff
@@ -134,6 +134,7 @@ public enum CodeElementType
         OnSizeErrorCondition,
         NotOnSizeErrorCondition,
         ElseCondition,
+        WhenDummy,
```

Comment:
Instead of creating a `WhenDummy` statement, it'd better to adapt `WhenCondition` in order to accept only `WHEN` keyword alone.
This way, there is not need to register a new CodeElement.

As stated #1835, this technique must be repeated on following statement, so it's better to find a minimalist approach:
- ACCEPT
- AT
- DECLARE -> unlikely to ask completion here
- DIVIDE
- GO
- ID -> unlikely to ask completion here
- IDENTIFICATION -> unlikely to ask completion here
- MULTIPLY
- SEARCH
- SUBTRACT
- USE -> USE DEBUGGING works. This is the only statement that starts with USE.

Reply:
Does this mean that 
whenSearchCondition 
should also accept only WHEN keyword alone?

Or this issue only concern whenCondition?